src/dev/src/app/exportable-datagrid/exportable-datagrid.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
+import { ExportTypeEnum } from '../../../../clr-addons/export-datagrid-button/export-datagrid-button.component';
```

Comment:
this is unnecessarily complicated, it should be something like ../export-datagrid-button